diff -urN kde-runtime-4.8.2.orig/kcontrol/componentchooser/componentchooser.desktop kde-runtime-4.8.2/kcontrol/componentchooser/componentchooser.desktop
--- kde-runtime-4.8.2.orig/kcontrol/componentchooser/componentchooser.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kcontrol/componentchooser/componentchooser.desktop	2012-04-13 01:16:18.124782000 +0400
@@ -161,7 +161,7 @@
 Comment[pt]=Escolha os componentes predefinidos para os vários serviços
 Comment[pt_BR]=Escolha os componentes padrão para vários serviços
 Comment[ro]=Alegeți componentele implicite pentru diferite servicii
-Comment[ru]=Выбор программ для основных задач
+Comment[ru]=Выбор приложений по умолчанию для различных служб
 Comment[se]=Vállje bálválusaid standárdosiid
 Comment[si]=විවිධ සේවාවන් වලට පෙරනිමි සංරචක පෙන්වන්න
 Comment[sk]=Vyberte štandardné komponenty pre rôzne služby
@@ -218,6 +218,7 @@
 X-KDE-Keywords[pt]=aplicações predefinidas,componentes,selector de componentes,recursos,e-mail,cliente de e-mail,editor de texto,mensageiro instantâneo,emulador de terminal,navegador Web,URL,hiperligações
 X-KDE-Keywords[pt_BR]=aplicativos padrão,componentes,seletor de componentes,recursos,e-mail,cliente de e-mail,editor de texto,mensageiro instantâneo,emulador de terminal,navegador web,URL,hiperligações
 X-KDE-Keywords[ro]=aplicații implicite,componente,alege componente,resurse,e-mail,email,client,editor text,mesager instant,emulator terminal, browser web,URL,link-uri
+X-KDE-Keywords[ru]=default applications,components,component chooser,resources,e-mail,email client,text editor,instant messenger,terminal emulator,web browser,URL,hyperlinks,приложения по умолчанию,компоненты,выбор компонентов,ресурсы,электронная почта,почтовый клиент,текстовый редактор,мгновенный обмен сообщениями,эмулятор терминала,веб-браузер,ссылка,гиперссылки,гиперссылка
 X-KDE-Keywords[sk]=predvolené aplikácie,komponenty,výber komponentu,zdroje,e-mail,emailový klient,textový editor,instant messenger,emulátor terminálu,webový prehliadač,URL,hyperlinky
 X-KDE-Keywords[sr]=default applications,components,component chooser,resources,e-mail,email client,text editor,instant messenger,terminal emulator,web browser,URL,hyperlinks,подразумевани програми,компонента,бирач компоненти,ресурс,е‑пошта,клијент е‑поште,уређивач текста,брзи гласник,емулатор терминала,веб прегледач,УРЛ,хипервеза
 X-KDE-Keywords[sr@ijekavian]=default applications,components,component chooser,resources,e-mail,email client,text editor,instant messenger,terminal emulator,web browser,URL,hyperlinks,подразумевани програми,компонента,бирач компоненти,ресурс,е‑пошта,клијент е‑поште,уређивач текста,брзи гласник,емулатор терминала,веб прегледач,УРЛ,хипервеза
diff -urN kde-runtime-4.8.2.orig/kcontrol/kded/kcmkded.desktop kde-runtime-4.8.2/kcontrol/kded/kcmkded.desktop
--- kde-runtime-4.8.2.orig/kcontrol/kded/kcmkded.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kcontrol/kded/kcmkded.desktop	2012-04-13 01:16:18.138782001 +0400
@@ -219,6 +219,7 @@
 X-KDE-Keywords[pt]=KDED,Servidor,Serviços
 X-KDE-Keywords[pt_BR]=KDED,Servidor,Serviços
 X-KDE-Keywords[ro]=KDED,Servicii,Demon
+X-KDE-Keywords=KDED,Daemon,Services,демон,службы,служба,сервисы
 X-KDE-Keywords[sk]=KDED,Démon,Služby
 X-KDE-Keywords[sr]=KDED,Daemon,Services,КДЕД,демон,сервис
 X-KDE-Keywords[sr@ijekavian]=KDED,Daemon,Services,КДЕД,демон,сервис
diff -urN kde-runtime-4.8.2.orig/kcontrol/knotify/kcmnotify.desktop kde-runtime-4.8.2/kcontrol/knotify/kcmnotify.desktop
--- kde-runtime-4.8.2.orig/kcontrol/knotify/kcmnotify.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kcontrol/knotify/kcmnotify.desktop	2012-04-13 01:16:18.136782002 +0400
@@ -201,6 +201,7 @@
 X-KDE-Keywords[pt]=Sons do sistema,Áudio,Som,Notificar,Alertas,Notificação,Mensagens
 X-KDE-Keywords[pt_BR]=Sons do sistema,Áudio,Som,Notificar,Alertas,Notificação,mensagens
 X-KDE-Keywords[ro]=Sunete sistem,audio,sunet,notificare,alerte,notifică,popup
+X-KDE-Keywords[ru]=System sounds,Audio,Sound,Notify,Alerts,Notification,popups,системные звуки,аудио,звук,уведомлять,сигналы,уведомления,всплывающее окно,всплывающие окна
 X-KDE-Keywords[sk]=Systémové zvuky,Audio,Zvuk,Notify,Alerts,Notification,popups
 X-KDE-Keywords[sr]=System sounds,Audio,Sound,Notify,Alerts,Notification,popups,системски звукови,аудио,звук,обавештење,аларм,искакач
 X-KDE-Keywords[sr@ijekavian]=System sounds,Audio,Sound,Notify,Alerts,Notification,popups,системски звукови,аудио,звук,обавештење,аларм,искакач
diff -urN kde-runtime-4.8.2.orig/kcontrol/locale/language.desktop kde-runtime-4.8.2/kcontrol/locale/language.desktop
--- kde-runtime-4.8.2.orig/kcontrol/locale/language.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kcontrol/locale/language.desktop	2012-04-13 01:16:18.093782001 +0400
@@ -223,6 +223,7 @@
 X-KDE-Keywords[pt]=língua,tradução,formato de números,localização,país,codificações,codificações de caracteres,símbolo decimal,separador de milhares,símbolo,separador,sinal,positivo,negativo,moeda,monetário,casas decimais,hora,data,formatos,semana,início da semana,primeiro,papel,tamanho,carta,A4,medida,medir,Inglês,Imperial
 X-KDE-Keywords[pt_BR]=idioma,tradução,formato de números,localização,país,codificações,codificações de caracteres,símbolo decimal,separador de milhares,símbolo,separador,sinal,positivo,negativo,moeda,monetário,casas decimais,calendário,hora,data,formatos,semana,início da semana,primeiro,papel,tamanho,carta,A4,medida,medir,Inglês,Imperial
 X-KDE-Keywords[ro]=limbă,traducere,format număr,localizare,țară,set de caractere,seturi de caractere,simbol decimal, separator de mii,simbol,separator,semn,pozitiv,negativ,monedă,bani,zecimale,calendar,timp,dată,formate, început,săptămînă,prima,hîrtie,dimensiune,A4,măsură,metric,unități,Imperial
+X-KDE-Keywords[ru]=language,translation,number format,locale,Country,charsets,character sets,Decimal symbol,Thousands separator,symbol,separator,sign,positive,negative,currency,money,fractional digits,calendar,time,date,formats,week,week start,first,paper,size,letter,A4,measure,metric,English,Imperial,язык,перевод,числовой формат,локаль,страна,кодировки,наборы символов,десятичный разделитель,разделитель групп разрядов,символ,разделитель,знак,положительный,отрицательный,текущий,деньги,дробная часть числа,календарь,время,дата,форматы,неделя,начало недели,первый,бумага,размер,письмо,А4,делитель,метрика,английский,императорский
 X-KDE-Keywords[sk]=jazyk,preklad,number format,locale,Krajina,charsets,character sets,Decimal symbol,Thousands separator,symbol,oddeľovač,znamienko,kladný,záporný,mena,peniaze,fractional digits,kalendár,čas,dátum,formáty,týždeň,week start,prvý,papier,veľkosť,letter,A4,measure,metric,English,Imperial
 X-KDE-Keywords[sr]=language,translation,number format,locale,Country,charsets,character sets,Decimal symbol,Thousands separator,symbol,separator,sign,positive,negative,currency,money,fractional digits,calendar,time,date,formats,week,week start,first,paper,size,letter,A4,measure,metric,English,Imperial,језик,превод,формат бројева,локалитет,држава,кодирање,децимални симбол,раздвајач хиљада,раздвајач,симбол,знак,позитивно,негативно,валута,новац,разломак,календар,време,датум,формати,седмица,почетак седмице,недеља,папир,величина,А4,летер,метрички,империјални
 X-KDE-Keywords[sr@ijekavian]=language,translation,number format,locale,Country,charsets,character sets,Decimal symbol,Thousands separator,symbol,separator,sign,positive,negative,currency,money,fractional digits,calendar,time,date,formats,week,week start,first,paper,size,letter,A4,measure,metric,English,Imperial,језик,превод,формат бројева,локалитет,држава,кодирање,децимални симбол,раздвајач хиљада,раздвајач,симбол,знак,позитивно,негативно,валута,новац,разломак,календар,време,датум,формати,седмица,почетак седмице,недеља,папир,величина,А4,летер,метрички,империјални
diff -urN kde-runtime-4.8.2.orig/kcontrol/spellchecking/spellchecking.desktop kde-runtime-4.8.2/kcontrol/spellchecking/spellchecking.desktop
--- kde-runtime-4.8.2.orig/kcontrol/spellchecking/spellchecking.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kcontrol/spellchecking/spellchecking.desktop	2012-04-13 01:16:18.091782002 +0400
@@ -221,6 +221,7 @@
 X-KDE-Keywords[pt]=Ortografia
 X-KDE-Keywords[pt_BR]=verificação ortográfica
 X-KDE-Keywords[ro]=Ortografie
+X-KDE-Keywords[ru]=Spell,орфография,правописание,проверка
 X-KDE-Keywords[sk]=Pravopis
 X-KDE-Keywords[sl]=Črkuj
 X-KDE-Keywords[sr]=spell,правопис,писање
diff -urN kde-runtime-4.8.2.orig/khelpcenter/htmlsearch/htmlsearch.desktop kde-runtime-4.8.2/khelpcenter/htmlsearch/htmlsearch.desktop
--- kde-runtime-4.8.2.orig/khelpcenter/htmlsearch/htmlsearch.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/khelpcenter/htmlsearch/htmlsearch.desktop	2012-04-13 01:16:17.707821776 +0400
@@ -224,6 +224,7 @@
 X-KDE-Keywords[pt]=Ajuda,HTML,Pesquisa,Índice,htdig,htsearch,htmerge,Âmbito,Linguagem
 X-KDE-Keywords[pt_BR]=Ajuda,HTML,Pesquisa,Índice,htdig,htsearch,htmerge,Âmbito,Linguagem
 X-KDE-Keywords[ro]=ajutor,HTML,căutare,caută,cuprins,htdig,htsearch,htmerge,limbă
+X-KDE-Keywords[ru]=Help,HTML,Search,Index,htdig,htsearch,htmerge,Scope,Language,помощь,справка,поиск,индекс,масштаб,область,язык
 X-KDE-Keywords[sk]=Pomoc,HTML,Hľadať,Index,htdig,htsearch,htmerge,Scope,Jazyk
 X-KDE-Keywords[sr]=Help,HTML,Search,Index,htdig,htsearch,htmerge,Scope,Language,помоћ,ХТМЛ,претрага,тражење,индекс,језик
 X-KDE-Keywords[sr@ijekavian]=Help,HTML,Search,Index,htdig,htsearch,htmerge,Scope,Language,помоћ,ХТМЛ,претрага,тражење,индекс,језик
diff -urN kde-runtime-4.8.2.orig/khelpcenter/kcmhelpcenter.desktop kde-runtime-4.8.2/khelpcenter/kcmhelpcenter.desktop
--- kde-runtime-4.8.2.orig/khelpcenter/kcmhelpcenter.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/khelpcenter/kcmhelpcenter.desktop	2012-04-13 01:16:17.704822732 +0400
@@ -165,7 +165,7 @@
 Comment[pt]=Configuração e geração do índice de pesquisa do centro de ajuda
 Comment[pt_BR]=Configuração e geração do índice de busca do centro de ajuda
 Comment[ro]=Generează și configurează indexul de căutare pentru Centrul de Ajutor
-Comment[ru]=Настройка индекса справочной системы KDE
+Comment[ru]=Создание и настройка индекса поиска справочного центра
 Comment[se]=Veahkkeguovddáža ohcanindeavssa heiveheapmi ja ráhkadeapmi
 Comment[si]=උදව් මධ්‍යස්ථාන සූචිය සැකසීම හා ජනනය
 Comment[sk]=Nastavenie a generovanie indexu pre pomocníka
@@ -221,6 +221,7 @@
 X-KDE-Keywords[pt]=khelpcenter,ajuda,índice,pesquisa
 X-KDE-Keywords[pt_BR]=khelpcenter,ajuda,índice,pesquisa
 X-KDE-Keywords[ro]=khelpcenter,centru,ajutor,cuprins,căutare
+X-KDE-Keywords[ru]=khelpcenter,help,index,search,справка,индекс,поиск
 X-KDE-Keywords[sk]=khelpcenter,help,index,hľadanie
 X-KDE-Keywords[sr]=khelpcenter,help,index,search,К‑центар-помоћи,помоћ,индекс,претрага,тражење
 X-KDE-Keywords[sr@ijekavian]=khelpcenter,help,index,search,К‑центар-помоћи,помоћ,индекс,претрага,тражење
diff -urN kde-runtime-4.8.2.orig/kioslave/cgi/kcmcgi/kcmcgi.desktop kde-runtime-4.8.2/kioslave/cgi/kcmcgi/kcmcgi.desktop
--- kde-runtime-4.8.2.orig/kioslave/cgi/kcmcgi/kcmcgi.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kioslave/cgi/kcmcgi/kcmcgi.desktop	2012-04-13 01:16:19.585782002 +0400
@@ -164,7 +164,7 @@
 Comment[pt]=Configuração do KIO slave de CGIs
 Comment[pt_BR]=Configurar o KIO slave de CGIs
 Comment[ro]=Configurează dispozitivul I/O CGI
-Comment[ru]=Настройка поддержки CGI
+Comment[ru]=Настройка CGI KIO slave
 Comment[se]=Heivet CGI-KIO-šláva
 Comment[si]=CGI KIO වහලා සකසන්න
 Comment[sk]=Nastavenie CGI KIO klienta
@@ -221,6 +221,7 @@
 X-KDE-Keywords[pt]=CGI,KIO,Slave,Localizações
 X-KDE-Keywords[pt_BR]=CGI,KIO,Slave,Localizações
 X-KDE-Keywords[ro]=CGI,KIO,Slave,Căi
+X-KDE-Keywords[ru]=CGI,KIO,Slave,Paths,подчинённый,вспомогательный,пути,путь
 X-KDE-Keywords[sk]=CGI,KIO,Slave,Paths
 X-KDE-Keywords[sr]=CGI,KIO,Slave,Paths,ЦГИ,У/И захват,путања
 X-KDE-Keywords[sr@ijekavian]=CGI,KIO,Slave,Paths,ЦГИ,У/И захват,путања
diff -urN kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/cpan.desktop kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/cpan.desktop
--- kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/cpan.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/cpan.desktop	2012-04-13 01:16:17.894786988 +0400
@@ -66,7 +66,7 @@
 Name[pt]=CPAN - Comprehensive Perl Archive Network
 Name[pt_BR]=CPAN - Rede Compreensiva de Arquivos de Perl
 Name[ro]=CPAN - Arhiva completă de programe Perl
-Name[ru]=CPAN — архив скриптов Perl
+Name[ru]=CPAN — всеобъемлющая сеть архивов Perl
 Name[se]=CPAN – Perl-vuorká: «Comprehensive Perl Archive Network»
 Name[si]=CPAN - විචක්‍ෂණ Perl සංරක්‍ෂක ජාලය
 Name[sk]=CPAN - Comprehensive Perl Archive Network
diff -urN kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/ctan.desktop kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/ctan.desktop
--- kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/ctan.desktop	2012-01-03 23:37:58.000000000 +0400
+++ kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/ctan.desktop	2012-04-13 01:16:18.009782002 +0400
@@ -66,7 +66,7 @@
 Name[pt]=CTAN - Comprehensive TeX Archive Network
 Name[pt_BR]=CTAN - Rede Compreensiva de Arquivos de TeX
 Name[ro]=CTAN - Arhiva completă de programe TeX
-Name[ru]=CTAN — архив TeX
+Name[ru]=CTAN — всеобъемлющая сеть архивов TeX
 Name[se]=CTAN – TeX-vuorká: «Comprehensive TeX Archive Network»
 Name[si]=CTAN - Comprehensive TeX Archive Network
 Name[sk]=CTAN - Comprehensive TeX Archive Network
diff -urN kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop
--- kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop	2012-01-03 23:37:58.000000000 +0400
+++ kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop	2012-04-13 01:16:17.808799909 +0400
@@ -63,7 +63,7 @@
 Name[pt]=Motor de Busca de Lista Telefónica
 Name[pt_BR]=Provedor de busca de guia telefônico
 Name[ro]=Căutare în cartea de telefoane
-Name[ru]=Telephonebook Search Provider
+Name[ru]=Провайдер поиска Telephonebook
 Name[se]=Ohcanmotor telefonkatalogaid várás
 Name[si]=දුරකථන පොත් සෙවුම් සැපයුම් කරු
 Name[sk]=Hľadanie v telefónnom zozname
diff -urN kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/rfc.desktop kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/rfc.desktop
--- kde-runtime-4.8.2.orig/kurifilter-plugins/ikws/searchproviders/rfc.desktop	2012-03-01 02:56:07.000000000 +0400
+++ kde-runtime-4.8.2/kurifilter-plugins/ikws/searchproviders/rfc.desktop	2012-04-13 01:16:17.933783727 +0400
@@ -64,7 +64,7 @@
 Name[pt]=Pedidos de Comentários do IETF
 Name[pt_BR]=Requisições de comentários IETF
 Name[ro]=RFC-uri IETF
-Name[ru]=Поиск по RFC
+Name[ru]=Запрос комментариев (RFC) IETF
 Name[se]=IETF Requests for Comments
 Name[si]=ප්‍රතිචාර සඳහා IETF ඉල්ලීම්
 Name[sk]=IETF Requests for Comments
diff -urN kde-runtime-4.8.2.orig/phonon/kded-module/phononserver.desktop kde-runtime-4.8.2/phonon/kded-module/phononserver.desktop
--- kde-runtime-4.8.2.orig/phonon/kded-module/phononserver.desktop	2012-01-18 22:40:46.000000000 +0400
+++ kde-runtime-4.8.2/phonon/kded-module/phononserver.desktop	2012-04-13 01:16:18.945782002 +0400
@@ -52,7 +52,7 @@
 Name[pt]=Política de Som
 Name[pt_BR]=Política de som
 Name[ro]=Politică de sunet
-Name[ru]=Звуковой сервер
+Name[ru]=Правила доступа к звуковым устройствам
 Name[si]=ශ්‍රව්‍ය ප්‍රතිපත්තිය
 Name[sk]=Politika zvuku
 Name[sl]=Pravila za zvok
diff -urN kde-runtime-4.8.2.orig/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop kde-runtime-4.8.2/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop
--- kde-runtime-4.8.2.orig/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop	2012-01-03 23:37:58.000000000 +0400
+++ kde-runtime-4.8.2/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop	2012-04-13 01:16:18.970782000 +0400
@@ -132,7 +132,7 @@
 Comment[pt]=Execução de JavaScript
 Comment[pt_BR]=Mecanismo JavaScript
 Comment[ro]=Executor JavaScript
-Comment[ru]=Модуль запуска, написанный на языке JavaScript
+Comment[ru]=Модуль запуска JavaScript
 Comment[si]=JavaScript Runner
 Comment[sk]=Spúšťač JavaScriptu
 Comment[sl]=Zaganjalnik javascripta
diff -urN kde-runtime-4.8.2.orig/platforms/win/kwinstartmenu/winstartmenu.desktop kde-runtime-4.8.2/platforms/win/kwinstartmenu/winstartmenu.desktop
--- kde-runtime-4.8.2.orig/platforms/win/kwinstartmenu/winstartmenu.desktop	2012-01-18 22:40:46.000000000 +0400
+++ kde-runtime-4.8.2/platforms/win/kwinstartmenu/winstartmenu.desktop	2012-04-13 01:16:19.661781999 +0400
@@ -57,7 +57,7 @@
 Name[pt]=Módulo do Menu Inicial do Windows no KDED
 Name[pt_BR]=Módulo do "Menu Inciar" do Windows para o KDED
 Name[ro]=Modul KDED Meniu Start Windows
-Name[ru]=Меню входа в Windows
+Name[ru]=Модуль меню «Старт» для KDED
 Name[si]=KDED Windows ආරම්භක මෙනු මොඩියුලය
 Name[sk]=KDED modul pre Štart menu Windows
 Name[sl]=Modul KDED za zagonski meni v Windows
@@ -143,7 +143,7 @@
 Comment[pt]=Suporte de ícone de atalho
 Comment[pt_BR]=Suporte ao ícone de atalho
 Comment[ro]=Suport pentru pictograme accelerator
-Comment[ru]=Показ значков сайтов и ссылок в KDE
+Comment[ru]=Поддержка иконок сокращений
 Comment[se]=Govašlávkestagaid doarjja
 Comment[si]=කෙටිමං අයිකන සහාය
 Comment[sk]=Podpora ikon skratiek
diff -urN kde-runtime-4.8.2.orig/solid-device-automounter/kded/device_automounter.desktop kde-runtime-4.8.2/solid-device-automounter/kded/device_automounter.desktop
--- kde-runtime-4.8.2.orig/solid-device-automounter/kded/device_automounter.desktop	2012-03-30 00:26:38.000000000 +0400
+++ kde-runtime-4.8.2/solid-device-automounter/kded/device_automounter.desktop	2012-04-13 01:16:19.670781999 +0400
@@ -51,7 +51,7 @@
 Name[pt]=Montagem Automática dos Dispositivos Removíveis
 Name[pt_BR]=Montagem automática de dispositivos removíveis
 Name[ro]=Montator automat de dispozitive amovibile
-Name[ru]=Подключение внешних носителей
+Name[ru]=Автоматическое монтирование внешних носителей
 Name[si]=ඉවත්කල හැකි මෙවලම ස්වයංව සවිකරනය
 Name[sk]=Automatické pripájanie vymeniteľných zariadení
 Name[sl]=Samodejno priklapljanje odstranljivih naprav
