diff -urN kde-runtime-4.11.3/kcontrol/locale/kcmlocale.cpp kde-runtime-4.11.3-patched/kcontrol/locale/kcmlocale.cpp
--- kde-runtime-4.11.3/kcontrol/locale/kcmlocale.cpp	2013-08-29 03:59:56.000000000 +1100
+++ kde-runtime-4.11.3-patched/kcontrol/locale/kcmlocale.cpp	2013-11-19 15:07:49.684091333 +1100
@@ -22,10 +22,13 @@
 
 #include "kcmlocale.h"
 
+#include <QDir>
+#include <QDBusInterface>
 #include <QListWidget>
 #include <QListWidgetItem>
 #include <QPrinter>
 #include <QTimer>
+#include <QLocale>
 
 #include <KAboutData>
 #include <KActionSelector>
@@ -642,12 +645,51 @@
     copyCalendarSettings( &m_userCalendarSettings, &m_kcmCalendarSettings );
 }
 
+void KCMLocale::saveI18nSettings()
+{
+    QString languages = m_kcmSettings.readEntry( "Language", QString() );
+
+    QStringList list = languages.split(':');
+
+    if( !list.size() )
+        return;
+
+    QString path = QDir::homePath() + "/.i18n";
+    QFile m_i18nUserSettings(path);
+
+    if (list.at(0).isEmpty()) {
+        m_i18nUserSettings.remove();
+    } else {
+        QLocale loc(list.at(0));
+        const QString value = loc.bcp47Name().replace('-', '_')  + ".UTF-8";
+
+        m_i18nUserSettings.open(QIODevice::WriteOnly);
+
+        m_i18nUserSettings.write("LC_TELEPHONE=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_CTYPE=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LANGUAGE=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_MONETARY=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_ADDRESS=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_COLLATE=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_NAME=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_NUMERIC=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_MEASUREMENT=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_TIME=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LANG=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_IDENTIFICATION=" + value.toAscii() + "\n");
+        m_i18nUserSettings.write("LC_MESSAGES=" + value.toAscii() + "\n");
+
+        m_i18nUserSettings.close();
+    }
+}
+
 void KCMLocale::save()
 {
     m_userConfig->sync();
 
     // rebuild the date base if language was changed
     if ( m_currentTranslations != m_kcmSettings.readEntry( "Language", QString() ) ) {
+        saveI18nSettings();
         KMessageBox::information(this, ki18n("Changed language settings apply only to "
                                             "newly started applications.\nTo change the "
                                             "language of all programs, you will have to "
@@ -655,6 +697,8 @@
                                 ki18n("Applying Language Settings").toString(m_kcmLocale),
                                 QLatin1String("LanguageChangesApplyOnlyToNewlyStartedPrograms"));
         KBuildSycocaProgressDialog::rebuildKSycoca(this);
+        QDBusInterface ksmServer( "org.kde.ksmserver", "/KSMServer", "org.kde.KSMServerInterface" );
+        ksmServer.call("logout", 1, 0, 3);
     }
 
     load();
diff -urN kde-runtime-4.11.3/kcontrol/locale/kcmlocale.h kde-runtime-4.11.3-patched/kcontrol/locale/kcmlocale.h
--- kde-runtime-4.11.3/kcontrol/locale/kcmlocale.h	2013-06-29 04:12:33.000000000 +1100
+++ kde-runtime-4.11.3-patched/kcontrol/locale/kcmlocale.h	2013-11-19 15:09:32.422743637 +1100
@@ -275,6 +275,9 @@
     void initBinaryUnitDialect();
     void setBinaryUnitDialect( int newValue );
 
+    //User i18n settings
+    void saveI18nSettings();
+
 private Q_SLOTS:
 
     void updateSample();
