diff -urN kde-runtime-4.11.0/kcontrol/componentchooser/componentchooser.desktop kde-runtime-4.11.0-patched/kcontrol/componentchooser/componentchooser.desktop
--- kde-runtime-4.11.0/kcontrol/componentchooser/componentchooser.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/kcontrol/componentchooser/componentchooser.desktop	2013-08-15 15:20:04.794904770 +1100
@@ -161,7 +161,7 @@
 Comment[pt]=Escolha os componentes predefinidos para os vários serviços
 Comment[pt_BR]=Escolha os componentes padrão para vários serviços
 Comment[ro]=Alegeți componentele implicite pentru diferite servicii
-Comment[ru]=Выбор программ для основных задач
+Comment[ru]=Выбор приложений по умолчанию для различных служб
 Comment[se]=Vállje bálválusaid standárdosiid
 Comment[si]=විවිධ සේවාවන් වලට පෙරනිමි සංරචක පෙන්වන්න
 Comment[sk]=Vyberte štandardné komponenty pre rôzne služby
diff -urN kde-runtime-4.11.0/khelpcenter/kcmhelpcenter.desktop kde-runtime-4.11.0-patched/khelpcenter/kcmhelpcenter.desktop
--- kde-runtime-4.11.0/khelpcenter/kcmhelpcenter.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/khelpcenter/kcmhelpcenter.desktop	2013-08-15 15:20:04.794904770 +1100
@@ -164,7 +164,7 @@
 Comment[pt]=Configuração e geração do índice de pesquisa do centro de ajuda
 Comment[pt_BR]=Configuração e geração do índice de busca do centro de ajuda
 Comment[ro]=Generează și configurează indexul de căutare pentru Centrul de Ajutor
-Comment[ru]=Настройка индекса справочной системы KDE
+Comment[ru]=Создание и настройка индекса поиска справочного центра
 Comment[se]=Veahkkeguovddáža ohcanindeavssa heiveheapmi ja ráhkadeapmi
 Comment[si]=උදව් මධ්‍යස්ථාන සූචිය සැකසීම හා ජනනය
 Comment[sk]=Nastavenie a generovanie indexu pre pomocníka
diff -urN kde-runtime-4.11.0/kioslave/cgi/kcmcgi/kcmcgi.desktop kde-runtime-4.11.0-patched/kioslave/cgi/kcmcgi/kcmcgi.desktop
--- kde-runtime-4.11.0/kioslave/cgi/kcmcgi/kcmcgi.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/kioslave/cgi/kcmcgi/kcmcgi.desktop	2013-08-15 15:20:04.794904770 +1100
@@ -164,7 +164,7 @@
 Comment[pt]=Configuração do KIO slave de CGIs
 Comment[pt_BR]=Configurar o KIO slave de CGIs
 Comment[ro]=Configurează dispozitivul I/O CGI
-Comment[ru]=Настройка поддержки CGI
+Comment[ru]=Настройка CGI KIO slave
 Comment[se]=Heivet CGI-KIO-šláva
 Comment[si]=CGI KIO වහලා සකසන්න
 Comment[sk]=Nastavenie CGI KIO klienta
diff -urN kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/cpan.desktop kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/cpan.desktop
--- kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/cpan.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/cpan.desktop	2013-08-15 15:20:04.795904790 +1100
@@ -66,7 +66,7 @@
 Name[pt]=CPAN - Comprehensive Perl Archive Network
 Name[pt_BR]=CPAN - Rede Compreensiva de Arquivos de Perl
 Name[ro]=CPAN - Arhiva completă de programe Perl
-Name[ru]=CPAN — архив скриптов Perl
+Name[ru]=CPAN — всеобъемлющая сеть архивов Perl
 Name[se]=CPAN – Perl-vuorká: «Comprehensive Perl Archive Network»
 Name[si]=CPAN - විචක්‍ෂණ Perl සංරක්‍ෂක ජාලය
 Name[sk]=CPAN - Comprehensive Perl Archive Network
diff -urN kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/ctan.desktop kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/ctan.desktop
--- kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/ctan.desktop	2013-08-08 01:59:56.000000000 +1100
+++ kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/ctan.desktop	2013-08-15 15:20:04.795904790 +1100
@@ -66,7 +66,7 @@
 Name[pt]=CTAN - Comprehensive TeX Archive Network
 Name[pt_BR]=CTAN - Rede Compreensiva de Arquivos de TeX
 Name[ro]=CTAN - Arhiva completă de programe TeX
-Name[ru]=CTAN — архив TeX
+Name[ru]=CTAN — всеобъемлющая сеть архивов TeX
 Name[se]=CTAN – TeX-vuorká: «Comprehensive TeX Archive Network»
 Name[si]=CTAN - Comprehensive TeX Archive Network
 Name[sk]=CTAN - Comprehensive TeX Archive Network
diff -urN kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop
--- kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop	2013-08-08 01:59:56.000000000 +1100
+++ kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/nl-telephone.desktop	2013-08-15 15:20:04.795904790 +1100
@@ -63,7 +63,7 @@
 Name[pt]=Motor de Busca de Lista Telefónica
 Name[pt_BR]=Provedor de busca de guia telefônico
 Name[ro]=Căutare în cartea de telefoane
-Name[ru]=Telephonebook Search Provider
+Name[ru]=Провайдер поиска Telephonebook
 Name[se]=Ohcanmotor telefonkatalogaid várás
 Name[si]=දුරකථන පොත් සෙවුම් සැපයුම් කරු
 Name[sk]=Hľadanie v telefónnom zozname
diff -urN kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/rfc.desktop kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/rfc.desktop
--- kde-runtime-4.11.0/kurifilter-plugins/ikws/searchproviders/rfc.desktop	2013-08-08 01:59:56.000000000 +1100
+++ kde-runtime-4.11.0-patched/kurifilter-plugins/ikws/searchproviders/rfc.desktop	2013-08-15 15:20:04.795904790 +1100
@@ -64,7 +64,7 @@
 Name[pt]=Pedidos de Comentários do IETF
 Name[pt_BR]=Requisições de comentários IETF
 Name[ro]=RFC-uri IETF
-Name[ru]=Поиск по RFC
+Name[ru]=Запрос комментариев (RFC) IETF
 Name[se]=IETF Requests for Comments
 Name[si]=ප්‍රතිචාර සඳහා IETF ඉල්ලීම්
 Name[sk]=IETF Requests for Comments
diff -urN kde-runtime-4.11.0/phonon/kded-module/phononserver.desktop kde-runtime-4.11.0-patched/phonon/kded-module/phononserver.desktop
--- kde-runtime-4.11.0/phonon/kded-module/phononserver.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/phonon/kded-module/phononserver.desktop	2013-08-15 15:20:04.795904790 +1100
@@ -54,7 +54,7 @@
 Name[pt]=Política de Som
 Name[pt_BR]=Política de som
 Name[ro]=Politică de sunet
-Name[ru]=Звуковой сервер
+Name[ru]=Правила доступа к звуковым устройствам
 Name[si]=ශ්‍රව්‍ය ප්‍රතිපත්තිය
 Name[sk]=Politika zvuku
 Name[sl]=Pravilnik za zvok
diff -urN kde-runtime-4.11.0/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop kde-runtime-4.11.0-patched/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop
--- kde-runtime-4.11.0/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/plasma/scriptengines/javascript/data/plasma-scriptengine-runner-javascript.desktop	2013-08-15 15:20:04.796904808 +1100
@@ -134,7 +134,7 @@
 Comment[pt]=Execução de JavaScript
 Comment[pt_BR]=Mecanismo JavaScript
 Comment[ro]=Executor JavaScript
-Comment[ru]=Модуль запуска, написанный на языке JavaScript
+Comment[ru]=Модуль запуска JavaScript
 Comment[si]=JavaScript Runner
 Comment[sk]=Spúšťač JavaScriptu
 Comment[sl]=Zaganjalnik JavaScripta
diff -urN kde-runtime-4.11.0/platforms/win/kwinstartmenu/winstartmenu.desktop kde-runtime-4.11.0-patched/platforms/win/kwinstartmenu/winstartmenu.desktop
--- kde-runtime-4.11.0/platforms/win/kwinstartmenu/winstartmenu.desktop	2013-08-06 23:44:01.000000000 +1100
+++ kde-runtime-4.11.0-patched/platforms/win/kwinstartmenu/winstartmenu.desktop	2013-08-15 15:20:04.796904808 +1100
@@ -58,7 +58,7 @@
 Name[pt]=Módulo do Menu Inicial do Windows no KDED
 Name[pt_BR]=Módulo do "Menu Inciar" do Windows para o KDED
 Name[ro]=Modul KDED Meniu Start Windows
-Name[ru]=Меню входа в Windows
+Name[ru]=Модуль меню «Старт» для KDED
 Name[si]=KDED Windows ආරම්භක මෙනු මොඩියුලය
 Name[sk]=KDED modul pre Štart menu Windows
 Name[sl]=Modul KDED za zagonski meni v Windows
@@ -144,7 +144,7 @@
 Comment[pt]=Suporte de ícone de atalho
 Comment[pt_BR]=Suporte ao ícone de atalho
 Comment[ro]=Suport pentru pictograme accelerator
-Comment[ru]=Показ значков сайтов и ссылок в KDE
+Comment[ru]=Поддержка иконок сокращений
 Comment[se]=Govašlávkestagaid doarjja
 Comment[si]=කෙටිමං අයිකන සහාය
 Comment[sk]=Podpora ikon skratiek
diff -urN kde-runtime-4.11.0/solid-device-automounter/kded/device_automounter.desktop kde-runtime-4.11.0-patched/solid-device-automounter/kded/device_automounter.desktop
--- kde-runtime-4.11.0/solid-device-automounter/kded/device_automounter.desktop	2013-07-10 09:47:02.000000000 +1100
+++ kde-runtime-4.11.0-patched/solid-device-automounter/kded/device_automounter.desktop	2013-08-15 15:20:04.796904808 +1100
@@ -56,7 +56,7 @@
 Name[pt]=Montagem Automática dos Dispositivos Removíveis
 Name[pt_BR]=Montagem automática de dispositivos removíveis
 Name[ro]=Montator automat de dispozitive amovibile
-Name[ru]=Подключение внешних носителей
+Name[ru]=Автоматическое монтирование внешних носителей
 Name[si]=ඉවත්කල හැකි මෙවලම ස්වයංව සවිකරනය
 Name[sk]=Automatické pripájanie vymeniteľných zariadení
 Name[sl]=Samodejno priklapljanje odstranljivih naprav
